airbyte-webapp/src/packages/cloud/views/credits/CreditsPage/components/UsagePerConnectionTable.tsx

```diff
@@ -142,7 +142,11 @@ const UsagePerConnectionTable: React.FC<UsagePerConnectionTableProps> = ({ credi
         accessor: "creditsConsumed",
         collapse: true,
         customPadding: { right: 0 },
-        Cell: ({ cell }: CellProps<FullTableProps>) => <UsageValue>{cell.value}</UsageValue>,
+        Cell: ({ cell }: CellProps<FullTableProps>) => (
+          <UsageValue>
```

Comment:
Could we maybe get rid of the styled component here while we're in this file? 👼

Reply:
I may as well.  Had punted it due to new designs coming, but that'll make those changes easier to implement/review so 🤷‍♀️

Reply:
Yep!  There's still some things with alignment that I'd like to tackle, but those are in the `<Table/>` component and don't belong in this PR.